app/talents/my-profile/page.tsx

```diff
@@ -1,240 +1,15 @@
-"use client";
+import MyProfile from "@/app/components/profile-form";
+import { Metadata } from "next";
 
-import { useRef, useState, FormEvent } from "react";
-
-import DragAndDropFile from "../../components/drag-and-drop-file";
-import { Button } from "../../components/button";
-
-interface FileData {
-  name: string;
-  type: string;
-  data: string | ArrayBuffer | null;
-}
-
-export default function MyProfile() {
-  const invoiceInputValue = useRef(null);
-
-  const [file, setFile] = useState<false | FileData>(false);
-  const [isRenderedPage, setIsRenderedPage] = useState<boolean>(true);
-
-  const handleSubmit = async (e: FormEvent<HTMLFormElement>) => {
-    e.preventDefault();
-
-    const formData = {
-      jobHeadline: e.currentTarget["job-headline"].value,
-      firstName: e.currentTarget["first-name"].value,
-      lastName: e.currentTarget["last-name"].value,
-      country: e.currentTarget.country.value,
-      city: e.currentTarget.city.value,
-      phoneCountryCode: e.currentTarget["phone-country-code"].value,
-      phoneNumber: e.currentTarget["phone-number"].value,
-      email: e.currentTarget.email.value,
-      telegram: e.currentTarget.telegram.value,
-      aboutWork: e.currentTarget["about-work"].value,
-    };
-
-    // TODO: POST formData to the server with fetch
-    const res = await fetch("/api/talents/my-profile", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify(formData),
-    });
-
-    const data = await res.json();
-
-    if (!res.ok) {
-      throw new Error(data.message || "Something went wrong!");
-    } else {
-      alert("Profile Saved!");
-    }
-  };
+export const metadata: Metadata = {
+  title: "My Profile | Talents Pages | GoodHive",
```

Comment:
`title: "My Profile | Talents | GoodHive",`

Reply:
done! I also changed on the others pages for consistency ...